.github/workflows/ci.yaml

```diff
@@ -13,6 +13,6 @@ on:
 jobs:
   lint_and_test:
     name: lint and test
-    uses: developerproductivity/logilica-cli/.github/workflows/python_lint_and_test.yaml@main
+    uses: developerproductivity/logilica-cli/.github/workflows/python_lint_and_test.yaml@dashboard-link-fixes
```

Comment:
This will be reverted back to `main` after approved.

Reply:
🎗️